optimum/exporters/tasks.py

```diff
@@ -105,6 +105,8 @@ class TasksManager:
             "audio-ctc": "AutoModelForCTC",
             "audio-xvector": "AutoModelForAudioXVector",
             "stable-diffusion": "StableDiffusionPipeline",
+            "visual-question-answering": "AutoModelForVisualQuestionAnswering",
```

Comment:
```suggestion
            "visual-question-answering": "VisualBertForQuestionAnswering",
```

We can get away with this for now (since visualbert is the only model supported for those tasks in the onnx export for now). You could do as well for the other tasks waiting for the AutoModel in transformers.